docs/GettingStartedDocs/Contributors/SGX1FLCGettingStarted.md

```diff
@@ -98,7 +95,7 @@ Test project /home/youradminusername/openenclave/build
 Total Test time (real) =  83.61 sec
 ```
 
-A clean pass of the above unitests run is an indication that your Open Enclave setup was successful. You can start playing with the Open Enclave samples after following the instructions in the "Install" section below to configure samples for building,
+A clean pass of the above unit tests run is an indication that your Open Enclave setup was successful. You can start playing with the Open Enclave samples after following the instructions in the "Install" section below to configure samples for building,
```

Comment:
Nit: I think a documentation point of view, we may want to be careful using terminology such as "playing". The reasoning is that if I as a customer want to "play" with something, it doesn't sound assertive as saying something along the lines of "using" or "successfully using".